srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/Pages/SoapLogs.php

```diff
@@ -0,0 +1,146 @@
+                                        $end = $data["pos"] + $maxPagination < $data["max_page"] && $diff > 0 ? $data["pos"] + $maxPagination : $data["max_page"];
+
+                                        $link = menu_page_url(ACORE_SLUG . '-soap-logs', false);
+                                        if ($data["username"]) {
+                                            $link .= "&username" . $data["username"];
+                                        }
+                                        if ($data["order_id"]) {
+                                            $link .= "&order_id" . $data["order_id"];
+                                        }
+                                        ?>
+
+                                        <li class="page-item disabled">
+                                        <?php if ($data["count"] > 0): ?>
+                                        <a class="page-link">Results <?php echo ($data["pos"] - 1) * $data["items"] + 1; ?> to <?php echo $data["pos"] != $data["max_page"] ? $data["pos"] * $data["items"] : $data["count"]; ?> from <?php echo $data["count"]; ?> </a>
+                                        <?php else: ?>
+                                            <a class="page-link">No results </a>
+                                        <?php endif; ?>
+                                        </li>
+
+                                        <?php
+                                        $link .= "&items=" . $data["items"];
+                                        if ($start > 1) {
+                                            echo "<li class=\"page-item\"><a class=\"page-link\" href=\"$link&pos=1\">1</a></li>";
+                                            if ($maxPagination <= $start) {
+                                                echo "<li class=\"page-item disabled\"><a class=\"page-link\">...</a></li>";
+                                            }
+                                        }
+                                        for ($i = $start; $i <= $end; $i++) {
+                                            $href = "$link&pos=$i";
+                                            $class = "";
+                                            if ($i == $data["pos"]) {
+                                                $href = "#";
+                                                $class = " active";
+                                            }
+                                            echo "<li class=\"page-item{$class}\"><a class=\"page-link\" href=\"{$href}\">$i</a></li>";
+                                        }
+                                        if ($end < $data["max_page"]) {
+                                            if ($data["max_page"] - 1 != $end) {
+                                                echo "<li class=\"page-item disabled\"><a class=\"page-link\">...</a></li>";
```

Comment:
I don't know in terms of performance if this is better or worse, but for readability and practice reason is better avoiding `\"`
```suggestion
                                                echo '<li class="page-item disabled"><a class="page-link">...</a></li>';
```